### PR TITLE
Adding information personal to alexisllamas file

### LINKED ISSUE #19: alexisllamas file should include which trains and beers I prefer and some miscellaneous
I´m going to add 

-Types of trains

-Brands of beer

-Why I hate you

### PR DESCRIPTION
**Pull request by alexismellamo**
This work is cool and I can close #19

### CODE REVIEW

File: alexisllamas.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+
+My favorite type of trains are:
+- Electric trains
+- Steam locomotive
+- Diesel train
+- Low-floor tram
+- Freight train
+- Container trains
+
+My favorite brans of beer are:
```
Comment (FerPerales) [inline alexisllamas.md]: I think this should say **brands**

**Comment by alexismellamo** [comment]
You`re cool

File: alexisllamas.md (hunk 1 above)
Comment (alexismellamo) [inline alexisllamas.md]: Thanks mate!

**Comment by diegoavalos95** [comment]
hi

**Pull request merged**